disconnected/mirroring/oc-mirror-migration-v1-to-v2.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oc-mirror-migration-v1-to-v2"]
+= Migration from oc-mirror plugin v1 to v2
+include::_attributes/common-attributes.adoc[]
+:context: oc-mirror-migration-v1-to-v2
+
+toc::[]
+
+The oc-mirror v2 plugin introduces major changes to image mirroring workflows. This guide provides step-by-step instructions for migration while ensuring compatibility with oc-mirror plugin v2.
+
+[IMPORTANT]
+====
+oc-mirror plugin v2 does not provide an automatic way to migrate an `ImageSetConfiguration` from oc-mirror plugin v1 to v2. You must manually update the configurations.
```

Comment:
If we can add here a link to the section explaining what changes in the ImageSetConfiguration , it could help out the user. WDYT?